rest-client/src/main/java/org/sagebionetworks/bridge/rest/AuthenticationHandler.java

```diff
@@ -54,7 +56,11 @@ public AuthenticationHandler(SignIn signIn, UserSessionInfoProvider userSessionI
         }
         tryCount++;
 
-        userSession = userSessionInfoProvider.retrieveSession(signIn);
+        try {
+            userSession = userSessionInfoProvider.retrieveSession(signIn);
+        } catch (ConsentRequiredException e) {
+            LOG.debug("Ignoring 412 during auto-authentication");
```

Comment:
You don't have to be consented to get the user's own participant record. Signing in at that point, if you were not signed in and not consented, it would return 412 (unless you have administrative roles, then it wouldn't). 

You should never hide this exception: for users with roles using administrative APIs, it isn't thrown (even on sign in), and for APIs where it is returned, they will never work until the user signs the consent. It should be thrown out of our core libraries... applications when they get it, should go into the consent system.

Another problem with hiding this is if we update the application, introducing a new consent, and signing everyone out, a call intercepted by this service would not do the right thing, and send the user to consent, if this is swallowed.